TrainingExtensions/tensorflow/src/python/aimet_tensorflow/quantizer_info.py

```diff
@@ -284,7 +331,16 @@ def compute_encoding(self, bitwidth: int, use_symmetric_encodings: bool) -> libp
         :return: Encoding
         """
         if not self._is_encoding_frozen:
-            encoding = self.tensor_quantizer.computeEncoding(bitwidth, use_symmetric_encodings, False, False)
+            encoding = []
+            if isinstance(self.tensor_quantizer, list):
+                for tensor_quantizer in self.tensor_quantizer:
+                    encoding_val = tensor_quantizer.computeEncoding(bitwidth, use_symmetric_encodings, False, False)
+                    if encoding_val:
+                        tensor_quantizer.isEncodingValid = True
+                    encoding.append(encoding_val)
+            else:
+                encoding.append(self.tensor_quantizer.computeEncoding(bitwidth, use_symmetric_encodings, False, False))
+                encoding = encoding[0]
         else:
             encoding = self.get_encoding()
 
```

Comment:
Which code actually uses the encodings returned here?

Reply:
The compute_encodings encodings get used by quantsim to set encodings for the quantizer.